### ISSUE TITLE
Support file version 5.40 and above

### ISSUE THREAD

**Issue by bitstreamout** [ahupp/python-magic#240]
New file version 5.40 counts pages in PDF files:

```
abuild@noether:~/rpmbuild/BUILD/python-magic-0.4.18> file --version
file-5.40
magic file from /etc/magic:/usr/share/misc/magic
abuild@noether:~/rpmbuild/BUILD/python-magic-0.4.18> file test/testdata/test.pdf 
test/testdata/test.pdf: PDF document, version 1.2, 2 pages
```

**Comment by ahupp** [comment]
Are you requesting that this output gets parsed into some structured metadata?

**Comment by bitstreamout** [comment]
> Are you requesting that this output gets parsed into some structured metadata?

I do not get this ... I've submitted the attached patch to our local build of (python-)python-magic package here
[fix-4-file-5.40.zip](https://github.com/ahupp/python-magic/files/6342188/fix-4-file-5.40.zip)

**Comment by ahupp** [comment]
python-magic does not include a copy of libmagic, it requires that is provided by the system.  So if you had libmagic 5.40 installed this new feature should just work.

**Comment by bitstreamout** [comment]
With libmagic 5.40 the python-magic test of its pdf test file simply fails and the patch in the attached zip does fix this.  Compare with bug https://bugzilla.opensuse.org/show_bug.cgi?id=1184881 ... if you think this is not required that simply close this issue.

**Comment by ahupp** [comment]
Ah, you mean the tests.  Thanks for letting me know!

**Comment by pombredanne** [comment]
@bitstreamout since your patch makes the test fails with earlier `file` versions, what about testing `startswith('PDF document, version 1.2') instead?
@ahupp is there a way in your test framework?

**Comment by ahupp** [comment]
@pombredanne I fixed those tests to handle both outputs in 0.4.23
